src/els_text_synchronization.erl

```diff
@@ -8,6 +8,9 @@
         ]).
 
 -export([ generate_diagnostics/1 ]).
+-ifdef(TEST).
```

Comment:
I prefer not to use this kind of macros. There is no problem exporting this function.